lua/lspconfig/server_configurations/bufls.lua

```diff
@@ -0,0 +1,33 @@
+local util = require 'lspconfig.util'
+
+local bin_name = 'bufls'
+local cmd = { bin_name, 'serve' }
+
+if vim.fn.has 'win32' == 1 then
+  cmd = { 'cmd.exe', '/C', bin_name, 'start' }
+end
+
+return {
+  default_config = {
+    cmd = cmd,
+    filetypes = { 'proto' },
+    root_dir = function(fname)
+      return util.root_pattern('buf.work.yaml', '.git')(fname)
```

Comment:
@rinx should we also include 'buf.yaml' here, for buf modules that don't use workspaces?

Reply:
@stefanvanburen Thanks for pointing it out. You're right!
I forgot to add it. 😓